app.js

```diff
@@ -1,6 +1,15 @@
 const express = require("express");
 const app = express();
-const { getCategories, getReviews } = require("./controllers/games.controllers");
+const {
+  getCategories,
+  getReviews,
+} = require("./controllers/games.controllers");
+
+app.get("/api", (req, res)=>{
+    fs.readFile("/endpoints.json", "utf8", function (err, data){
```

Comment:
This looks. a bit funky. Please put it in a controller and use error handlers as I think this wouldn't feedback the errors to the client when using throw